packages/documentation/src/pages/forms/available-features-and-usage-guidelines.mdx

```diff
@@ -438,15 +439,22 @@ preSubmitInfo: {
   field: 'privacyAgreementAccepted',
   label: <span>I have read and accept the <a href="/privacy">privacy policy</a>.</span>,
   error: 'You must accept the privacy policy before continuing',
+  // optional override
+  CustomComponent: CustomPreSubmitComponent
 }
 ```
-
+### Default `PrivacyAgreement`
 ![Required checkbox before form submission](../../images/forms/Required-Checkbox.jpg)
 
+### `CustomComponent` example
+![Required checkbox before form submission](../../images/forms/preSubmitInfo-customComponent.jpeg)
+
 ### Usage guidelines
 
 If you don't specify a `preSubmitInfo` section, no notice or checkbox appears above the submit button. Most applications will want to give some sort of notice to the user before they submit the form. Although this section is optional, we recommend you specify it.
 
+`sectionCompleted` is a function which expects a single boolean parameter indicating when all conditions of your `preSubmitInfo.CustomComponent` are met. Until `sectionCompleted` is called with `true`, the form will not submit.
```

Comment:
Might just be my OCD, but I like eliminating any chance of confusion. I'd rewrite the beginning of the line to say: 
"`sectionCompleted` is a function provided by `preSubmitInfo` which expects a single boolean parameter..."